backend/src/main/java/com/bibliotheek/config/ExternalPropertyConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.bibliotheek.config;
+
+import java.util.Optional;
+
+public final class ExternalPropertyConfig {
+
+    private ExternalPropertyConfig() {
+        // private constructor
+    }
+
+    public static void loadExternalProperties(String property) {
+        consoleLog("********************************************************");
+
+        String config = Optional.ofNullable(System.getProperty(property)).orElseGet(() -> System.getenv(property));
+
+        if (config != null && !config.trim().equals("")) {
+            consoleLog("lookup: " + property + " -> '" + config + "'");
+            consoleLog("using properties from : '" + config + "'");
+            System.setProperty("spring.config.location", config);
+        } else {
+            consoleLog("using properties from war");
+        }
+
+        consoleLog("********************************************************");
+    }
+
+    /**
+     * This method logs to system.out because the property loading must be the first thing the application does and therefore real
+     * logging is not loaded yet.
+     *
+     * @param logLine
```

Comment:
always add tag description beside the variable name in tag, i.e @param logline  Log Line